docs/source/getting_started/Installing.rst

```diff
@@ -237,3 +237,129 @@ se, and choose the conda file saved earlier with the Spark session configuration
     - numpy
     - pip:
         - msticpy[azure]>=2.3.1
+
+Installing for isolated or semi-isolated environments
+-----------------------------------------------------
+
+There are instances in which you may want to utilize msticpy in an isolated or semi-isolated environment. It is recommended to utilize a virtual environment and the scripts provided within the steps. Also ensure you have the correct python version installed on your host machine as well as the isolated environment. In order to find the correct python version, you can run the following:
+
```

Comment:
It would be good to preface with the fact that there are 2 methods:
- using manual pip commands (which is applicable when your source and target environments are the same)
- using the scripts is simpler, guaranteed more isolation and essential when building for cross-architecture.

Reply:
added